e2e/pageobjects/ide/Editor.ts

```diff
@@ -296,6 +298,39 @@ export class Editor {
         return lineYCoordinate;
     }
 
+    async clickOnTheTextInDefinedPosition(line: number, column: number) {
+        const additionalShiftingToY = 19;
+        const additionalShiftingToX = 1;
+        const yPosition: number = await this.getLineYCoordinates(line) + additionalShiftingToY;
+        const xPosition: number = column + additionalShiftingToX;
+        new  ActionSequence(this.driverHelper.getDriver()).
```

Comment:
please, add empty lines to make the method more readable.

Reply:
added